### PR TITLE
Added basic detection for packetlogic devices

### PR DESCRIPTION
**Pull request by laf**
I'll email the company for the logo - until then. this is using the linux logo.

### CODE REVIEW

**Comment by f0o** [comment]
Is there a typo in this filename?
`includes/polling/os/procera.inc..php`

**Comment by laf** [comment]
erm no :)

You can leave this one for a bit if you like. Waiting on someone to test it which shouldn't be long.

**Comment by Lonecrowe** [comment]
I tried to rediscover our procera devices and it didn't pick it up.  Still generic.

**Comment by laf** [comment]
Hi @Lonecrowe 

It's not in the main branch so you would need to switch to my testing one. Are you able to do that?

**Comment by f0o** [comment]
@laf Any news on the logo?

File: includes/definitions.inc.php [hunk 1]
```diff
@@ -140,6 +140,18 @@
 $config['os'][$os]['over'][2]['graph']  = "device_mempool";
 $config['os'][$os]['over'][2]['text']   = "Memory Usage";
 
+$os = "procera";
+$onfig['os'][$os]['type']              = "network";
```
Comment (f0o) [inline includes/definitions.inc.php]: typo :)

**Comment by laf** [comment]
@f0o Fixed missing c.

No news on logo, I'm not expecting to hear back.

**Comment by laf** [comment]
@Lonecrowe do the following:

cd /opt/librenms
git remote add laf https://github.com/laf/librenms.git
git remote update
git checkout issue-746

./discovery.php -h HOSTNAME -d -m os
./poller.php -h HOSTNAME -d -r -m system,os

If it works, groovy. If not paste the output from those somewhere (edit the results).

Switch back to master:

git checkout master

**Comment by Lonecrowe** [comment]
Followed all the instructions.. no dice. 

Here it the discovery of one of the devices
http://pastebin.com/7CABfGAx

Poller 
http://pastebin.com/zCPyE9SB

**Comment by laf** [comment]
This is ready now if someone can review.

**Pull request merged**